### PR TITLE
For Easy Config with Config-ui-x

### PR DESCRIPTION
**Pull request by donavanbecker**
@alessandroaime for easy config with [config-ui-x](https://github.com/oznu/homebridge-config-ui-x)

### CODE REVIEW

**Comment by alessandroaime** [comment]
I was just about to start working on it. Great job @donavanbecker!

I'll test it to make sure everything is working and then merge it. Thanks for the contribution!

**Comment by donavanbecker** [comment]
Awesome. Thanks!

**Comment by donavanbecker** [comment]
Just made one extra change to the pattern of the mac address.

**Comment by alessandroaime** [comment]
Just a small "mistake" in the regex checking the speaker, it throws a visual error when capital letters are inserted. You can fix it by changing it to `^([a-fA-F0-9]{2}:){5}[a-fA-F0-9]{2}$`.

Screenshot 2020-01-30 at 21 49 45

File: config.schema.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            },
+            "accessory": {
+                "title": "Accessory",
+                "type": "string",
+                "default": "UEBoomSpeaker",
+                "required": true,
+                "description": "This must be UEBoomSpeaker."
+            },
+            "speaker": {
+                "title": "speaker",
```
Comment (alessandroaime) [inline config.schema.json]: Also, if you could fix the `title` to `Speaker` it would be great.

Comment (donavanbecker) [reply config.schema.json]: made the Speaker uppercase.

**Comment by donavanbecker** [comment]
> Just a small "mistake" in the regex checking the speaker, it throws a visual error when capital letters are inserted. You can fix it by changing it to `^([a-fA-F0-9]{2}:){5}[a-fA-F0-9]{2}$`.
> 
> Screenshot 2020-01-30 at 21 49 45

I changed the pattern to what you suggested.

**Comment by donavanbecker** [comment]
Thanks!

**Pull request merged**